Packs/Kenna/Integrations/KennaV2/KennaV2.py

```diff
@@ -551,51 +582,107 @@ def delete_tags(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List[D
+    context: list[dict[str, Any]] = parse_response(assets_list, wanted_keys, actual_keys)
+    for lst in assets_list:
+        human_readable.append({
+            'id': lst.get('id'),
+            'Hostname': lst.get('hostname'),
+            'IP-address': lst.get('ip_address'),
+            'Vulnerabilities Count': args.get('vulnerabilities_count'),
+            'Operating System': lst.get('operating_system'),
+            'Score': lst.get('risk_meter_score')
+        })
+
+    return CommandResults(
+        outputs_prefix="Kenna.Assets",
+        outputs_key_field="ID",
+        readable_output=tableToMarkdown('Kenna Assets', human_readable, removeNull=True),
+        outputs=context if to_context else None,
+        raw_response=response
+    )
 
 
 def main():
+    command = demisto.command()
     params = demisto.params()
+    args = demisto.args()
+
     api = params.get('credentials_key', {}).get('password') or params.get('key')
     if not api:
         raise DemistoException('Kenna API key must be provided.')
     # Service base URL
-    base_url = params.get('url')
+    base_url = params.get('url', '')
     # Should we use SSL
     use_ssl = not params.get('insecure', False)
     # Should we use system proxy settings
-    use_proxy = params.get('proxy')
+    use_proxy = params.get('proxy', False)
     # Initialize Client object
     client = Client(base_url=base_url, api_key=api, verify=use_ssl, proxy=use_proxy)
-    command = demisto.command()
-    LOG(f'Command being called is {command}')
-    # Commands dict
-    commands: Dict[str, Callable[[Client, Dict[str, str]], Tuple[str, Dict[Any, Any], List[Any]]]] = {
+
+    demisto.debug(f'Command being called is {command}')
+
+    commands: dict[str, Callable[[Client, dict[str, str]], tuple[str, dict[Any, Any], list[Any]]]] = {
         'test-module': test_module,
         'kenna-search-vulnerabilities': search_vulnerabilities,
         'kenna-get-connectors': get_connectors,
         'kenna-run-connector': run_connector,
         'kenna-search-fixes': search_fixes,
-        'kenna-update-asset': update_asset,
         'kenna-update-vulnerability': update_vulnerability,
-        'kenna-search-assets': search_assets,
         'kenna-get-asset-vulnerabilities': get_asset_vulnerabilities,
         'kenna-add-tag': add_tags,
         'kenna-delete-tag': delete_tags,
         'kenna-get-connector-runs': get_connector_runs
     }
-
     try:
         if command in commands:
-            return_outputs(*commands[command](client, demisto.args()))
+            return_outputs(*commands[command](client, args))
+        elif command == "kenna-update-asset":
+            return_results(update_asset_command(client, args))
+        elif command == "kenna-search-assets":
+            return_results(search_assets_command(client, args))
+        elif command == "kenna-search-assets-by-external-id":
+            return_results(search_assets_by_external_id_command(client, args))
```

Comment:
use a dictionary instead of repeated `if`
`if command in ... : return_results(...)`